src/ui/components/component.js

```diff
@@ -367,12 +369,22 @@ export default class Component {
     // Process the DOM to determine if we should create
     // in-memory sub-components for rendering
     const domComponents = DOM.queryAll(this._container, '[data-component]:not([data-is-component-mounted])');
-    const data = this.transformData(cloneDeep(this._state.get()));
+    const data = this.transformData
+      ? this.transformData(cloneDeep(this._state.get()))
+      : this._state.get();
     domComponents.forEach(c => this._createSubcomponent(c, data));
 
-    this._children.forEach(child => {
-      child.mount();
-    });
+    if (this._progressivelyRenderChildren) {
+      this._children.forEach(child => {
+        setTimeout(() => {
+          child.mount();
+        }, 0);
```

Comment:
I think the default is 0, but don't mind specifying it! I feel a little split whether setTimeout defaulting to 0 is something people should just know by heart, which probably means that it's not

Reply:
ya, this could go either way